uk.co.compucorp.civicrm.hrleaveandabsences/js/src/leave-absences/leave-type-wizard/components/leave-type-wizard.component.js

```diff
@@ -246,18 +271,26 @@ define([
      * @param {Number} sectionIndex
      */
     function openSection (sectionIndex) {
-      var sectionToOpen = vm.sections[sectionIndex];
+      var sectionToOpen;
 
-      vm.sections.forEach(function (section) {
-        section.active = false;
-      });
+      if (sectionIndex === -1) {
+        navigateToLeaveTypesList();
+
+        return;
+      }
+
+      sectionToOpen = vm.sections[sectionIndex];
 
       if (!sectionToOpen) {
-        save();
+        submit();
```

Comment:
I'm guessing that this will submit the form when you reach the last section and there's nothing else to do.

I have a bit of an issue here, again, this seems that this needs to be inside a `openNextSection` method:


```js
function openPreviousSection () {
  var isOnFirstSection = currentSectionIndex === 0;
  
  if (isOnFirstSection) {
    navigateToLeaveTypesList();

    return;
  }

  currentSectionIndex--;

  openSection(currentSectionIndex);
}

function openNextSection () {
  var isOnLastSection = currentSectionIndex === vm.sections.length - 1;

  if (isOnLastSection) {
    submit(); // submitWizardForm would be nice 🤷‍♂️ 
  }

  currentSectionIndex++;

  openSection(currentSectionIndex);
}
```

Reply:
Done.